src/field_message.c

```diff
@@ -0,0 +1,99 @@
+#include "field_message.h"
+
+#include <nitro.h>
+#include <string.h>
+
+#include "bg_window.h"
+#include "font.h"
+#include "game_options.h"
+#include "render_text.h"
+#include "render_window.h"
+#include "strbuf.h"
+#include "text.h"
+
+void FieldMessage_LoadTextPalettes(u32 palLocation, u32 resetPrinters)
+{
+    if (resetPrinters == TRUE) {
+        Text_ResetAllPrinters();
+    }
+
+    Font_LoadTextPalette(palLocation, 13 * 32, HEAP_ID_FIELD);
+    Font_LoadScreenIndicatorsPalette(palLocation, 12 * 32, HEAP_ID_FIELD);
```

Comment:
**suggestion**: These should match with the `PLTT_OFFSET` macro (in `include/palette.h`) on the first operand:

```suggestion
    Font_LoadTextPalette(palLocation, PLTT_OFFSET(13), HEAP_ID_FIELD);
    Font_LoadScreenIndicatorsPalette(palLocation, PLTT_OFFSET(12), HEAP_ID_FIELD);
```

It also seems like we have a `FIELD_MESSAGE_PALETTE_INDEX` constant in `include/constants/field/window.h`...? Interesting!

Reply:
[`af167e0` (#326)](https://github.com/pret/pokeplatinum/pull/326/commits/af167e0ac8a74a3eb9c473abfc6424454840583b)

yep, it does match!